### ISSUE TITLE
Database users~logger is not created

### ISSUE THREAD

**Issue by felipeZ** [MD-Studio/MDStudio#116]
I have being trying to start the infrastructure like
```
13:57:00 $ ./start.sh 
Creating network "mdstudio_default" with the default driver
Creating mdstudio_redis_1 ... 
Creating mdstudio_mongo_1 ... 
Creating mdstudio_redis_1
Creating mdstudio_mongo_1 ... done
Creating mdstudio_crossbar_1 ... 
Creating mdstudio_crossbar_1 ... done
```
Then when I look into the logs I get:
```
13:57:07 $ docker logs mdstudio_crossbar_1                                                               
Loading .env environment variables…
....
2018-03-28T11:57:10+0000 [Controller      7] Local node configuration applied successfully!
2018-03-28T11:57:10+0000 [Router         20] Uploaded schemas for AuthComponent
2018-03-28T11:57:10+0000 [Router         20] Creating database "users~auth"
2018-03-28T11:57:10+0000 [Router         20] Creating collection users in users~auth
2018-03-28T11:57:11+0000 [Router         20] Uploaded schemas for LoggerComponent
2018-03-28T11:57:11+0000 [Router         20] Creating database "users~logger"
2018-03-28T11:57:11+0000 [Router         20] LoggerComponent: 3 procedures successfully registered
2018-03-28T11:57:11+0000 [Router         20] Creating collection groups~mdstudio in users~logger
2018-03-28T11:57:11+0000 [Router         20] Uploaded schemas for CacheComponent
2018-03-28T11:57:11+0000 [Router         20] CacheComponent: 6 procedures successfully registered
2018-03-28T11:57:11+0000 [Router         20] Creating collection groups in users~auth
2018-03-28T11:57:11+0000 [Router         20] SchemaComponent: 2 procedures successfully registered
2018-03-28T11:57:11+0000 [Router         20] Uploaded schemas for DBComponent
2018-03-28T11:57:11+0000 [Router         20] DBComponent: 20 procedures successfully registered
```
But when querying the `mongodb` the **users~logger database** is not created
```
In [1]: from pymongo import MongoClient

In [2]: client = MongoClient(
   ...: host='localhost', port=27017, serverSelectionTimeoutMS=1)

In [3]: client.database_names()
Out[3]: ['admin', 'config', 'local', 'users~auth', 'users~db', 'users~schema']
```

**Comment by felipeZ** [comment]
I have this [MD](https://github.com/MD-Studio/MDStudio/blob/v1/components/lie_md/lie_md/wamp_services.py#L19) class that is a subclass of **ComponentSession** and I trying to store something in the database using [this function](https://github.com/MD-Studio/MDStudio/blob/v1/components/lie_md/lie_md/cerise_interface.py#L40) but I there is nothing in the database and the function is never returns

**Comment by felipeZ** [comment]
fixed by 07f7bb5
